### DISCUSSION TITLE
Autocomplete multiple dropdown

### DISCUSSION THREAD

**Discussion by Eggwise** [directus/directus#18798] [Feature Requests]
### Summary

The autocomplete via api is very nice but it would be nice to have a multiple option to save values in a json array just like dropdown. Now i can only use the autocomplete for 40% of the fields i want.

### Basic Example

-

### Motivation

-

### Detailed Design

-

### Requirements List

-

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

-

**Comment by br41nslug** [comment]
Can you elaborate on your suggestion? For instance:
How would this multi-autocomplete work in the interface? It can't put the checkboxes inside the dropdown because the results may not be finite. How would you de-select items? etc.

**Reply by Eggwise** [reply]
Thanks for your reaction.

Just the same as autocomplete but then with multiple values.

https://www.google.com/search?q=multiple+select+dropdown+searc&oq=multiple+select+dropdown+search

**Comment by rijkvanzanten** [comment]
Please fill out the whole form 👍🏻 There's not enough information here to review or discuss
